### PR TITLE
Add Custom Color for Events, Add Teamchat Color

### PR DESCRIPTION
**Pull request by FaiThiX**
Tested on almost every game event, no errors detected.
If there is something to change pls let me know

This PR includes:
Teamchat Color Setting in util/constants.js
Event Color Setting for Discord Embed in util/constants.js
Embedicon changed to server image in discordTools/discordEmbeds.js

### CODE REVIEW

**Comment by mokimo5896** [comment]
suugestion : https://github.com/alexemanuelol/rustplusplus/pull/159  please add bot

File: src/discordTools/discordMessages.js [hunk 1]
```diff
@@ -385,7 +385,11 @@ module.exports = {
         const instance = Client.client.getInstance(guildId);
 
         let color = null;
-        if (message.color.length === 4) {
+        //Check if COLOR_TEAMCHAT is not null and 7 digits long, what represent a Hex Color Code. e.g. #ffffff
```
Comment (alexemanuelol) [inline src/discordTools/discordMessages.js:388]: Use /**/

File: src/util/constants.js [hunk 2]
```diff
@@ -56,5 +56,38 @@ module.exports = {
     COLOR_ACTIVE: '#00ff40',
     COLOR_INACTIVE: '#ff0040',
     COLOR_SETTINGS: '#861c0c',
-    COLOR_GREY: '#606060'
+    COLOR_GREY: '#606060',
+    COLOR_TEAMCHAT: '#50C878',
```
Comment (alexemanuelol) [inline src/util/constants.js:60]: Perhaps set this to default null so it uses rust+ teamchat color coding.

**Comment by alexemanuelol** [comment]
I hope the colors are pretty ;-)

**Comment by FaiThiX** [comment]
added your suggestions @alexemanuelol

**Pull request merged**